CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog for Qandidate toggle API
+
+## 0.3.x
+
+- Replaced `igorw/config-service-provider` with `vlucas/phpdotenv`. With this method there is [more flexibility to deploy] the API
+  `.env.dist` gives you the same configuration as `config.json.dist` gave you before.
```

Comment:
The config.json's `redis_uri` is replaced with `TOGGLE__REDIS_DSN`, aka `uri` with `dsn`. Might be worth mentioning!